specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2018-01-01/examples/ApiManagementListSKUs.json

```diff
@@ -0,0 +1,65 @@
+{
+	"parameters": {
+		"api-version": "2018-01-01",
+		"serviceName": "apimService1",
+		"resourceGroupName": "rg1",
+		"subscriptionId": "subid"
+	},
+	"responses": {
+		"200": {
+			"body": {
+				"value": [
+					{
+						"resourceType": "Microsoft.ApiManagement/service",
+						"sku": {
+							"name": "Developer"
+						},
+						"capacity": {
+							"minimum": 1,
+							"maximum": 1,
+							"default": 1,
+							"scaleType": "none"
+						}
+					},
+					{
+						"resourceType": "Microsoft.ApiManagement/service",
+						"sku": {
+							"name": "Basic"
+						},
+						"capacity": {
+							"minimum": 1,
+							"maximum": 2,
+							"default": 1,
+							"scaleType": "manual"
+						}
+					},
+					{
+						"resourceType": "Microsoft.ApiManagement/service",
+						"sku": {
+							"name": "Standard"
+						},
+						"capacity": {
+							"minimum": 1,
+							"maximum": 4,
+							"default": 1,
+							"scaleType": "automatic"
+						}
+					},
+					{
+						"resourceType": "Microsoft.ApiManagement/service",
+						"sku": {
+							"name": "Premium"
+						},
+						"capacity": {
+							"minimum": 1,
+							"maximum": 10,
+							"default": 1,
+							"scaleType": "automatic"
+						}
+					}
+				],
```

Comment:
In general the casing of enum enum values in examples must match with the actual enum values in swagger. Please double check 'scaleType' property values in this example.